test/compress/arrays.js

```diff
@@ -8,7 +8,7 @@ holes_and_undefined: {
     expect: {
         w=[1,,];
         x=[1,2,void 0];
-        y=[1,,2];
+        y=[1,,2,];
```

Comment:
Ref: https://tc39.github.io/ecma262/#sec-array-initializer

"If an element is elided at the end of an array, that element does not contribute to the length of the Array."